<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TopologyGeometryResolverLayerOptionsWidget</class>
 <widget class="QWidget" name="TopologyGeometryResolverLayerOptionsWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>402</width>
    <height>200</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QWidget" name="draw_style_placeholder_widget" native="true"/>
   </item>
   <item>
    <widget class="QCheckBox" name="fill_polygons">
     <property name="text">
      <string>Fill polygons</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="fill_opacity_intensity_layout">
     <item>
      <widget class="QLabel" name="fill_opacity_label">
       <property name="text">
        <string>Fill opacity:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QDoubleSpinBox" name="fill_opacity_spinbox">
       <property name="keyboardTracking">
        <bool>false</bool>
       </property>
       <property name="maximum">
        <double>1.000000000000000</double>
       </property>
       <property name="singleStep">
        <double>0.040000000000000</double>
       </property>
       <property name="value">
        <double>1.000000000000000</double>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="fill_intensity_label">
       <property name="text">
        <string>Fill intensity:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QDoubleSpinBox" name="fill_intensity_spinbox">
       <property name="keyboardTracking">
        <bool>false</bool>
       </property>
       <property name="maximum">
        <double>1.000000000000000</double>
       </property>
       <property name="singleStep">
        <double>0.040000000000000</double>
       </property>
       <property name="value">
        <double>1.000000000000000</double>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../qt-resources/qt_resources.qrc"/>
  <include location="../qt-resources/qt_resources.qrc"/>
 </resources>
 <connections/>
</ui>
